### DISCUSSION TITLE
Bluetooth support

### DISCUSSION THREAD

**Discussion by jscancado** [onlaj/Piano-LED-Visualizer#325] [Q&A]
Hi, I would like to adapt it to my digital accordion and I don’t to be attached to cables.

I would like to know if I could use a Yamaha MD-BT01 Bluetooth Wireless MIDI Adaptor, would that work? 
Would I need any additional hardware or software update?

thank you all!

**Comment by JeaAnny** [comment]
If your bluetooth organ supports the voltage rating of this [adapter](www.baudcom.cn), that's fine.
